packages/element/src/test/serialize.js

```diff
@@ -322,12 +322,11 @@ describe( 'renderElement()', () => {
 				<Provider value={ { value: '1st provided' } }>
 					<Consumer>{ ( context ) => context.value }</Consumer>
 				</Provider>
-				{ '|' }
+				|
 				<Provider value={ { value: '2nd provided' } }>
 					<Consumer>{ ( context ) => context.value }</Consumer>
 				</Provider>
-				{ '|' }
-				<Consumer>{ ( context ) => context.value }</Consumer>
+				|<Consumer>{ ( context ) => context.value }</Consumer>
```

Comment:
Should these `|` have been on separate lines?

Reply:
I was also a bit weirded out by this, and ran a full pass of formatting just in case. Seems to be just a bit of a weird edge case, but it conforms with auto-formatting so 🤷‍♂️

Reply:
cc @jsnajdr: Prettier is being unpretty here 😅

Reply:
I need some more info about what exactly is the problem. Prettier didn't do the change from `{ '|' }` to `|`, did it? It was done manually to fix (or autofix) the curly brace rule, right?

Is the new syntax incorrect? Does it produce, for example, more or less whitespace that expected in the rendered HTML?

Or is it just unpleasant to look at?

Reply:
Looks like it was auto-fixed but I'll let @DaniGuardiola confirm. 

Looks unpleasant to me, but I'm happy to just not look at it too much. 😅